python/dgl/distributed/partition.py

```diff
@@ -334,6 +334,21 @@ def load_partition(part_config, part_id, load_feats=True, use_graphbolt=False):
         "part-{}".format(part_id) in part_metadata
     ), "part-{} does not exist".format(part_id)
     part_files = part_metadata["part-{}".format(part_id)]
+
+    if use_graphbolt is False:
+        if os.path.exists(
+            os.path.join(config_path, f"part{part_id}", "graph.dgl")
+        ):
+            use_graphbolt = False
+        elif os.path.exists(
+            os.path.join(
+                config_path, f"part{part_id}", "fused_csc_sampling_graph.pt"
+            )
+        ):
+            use_graphbolt = True
```

Comment:
this may violate user's intent if both `graph.dgl`and `fused_xxx.pt` exists. As the existence of both of them is a previous result and it should not happen since now, so it's acceptable here. but in case, let's just crash here if this case happens, just ask user to remove one of them.